ci/deps.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+set -eu
+
+deps_alpine() {
+  apk upgrade
+
+  apk add \
+    git binutils make autoconf automake gcc linux-headers diffutils texinfo \
+    procps socat shadow sudo openssl-dev zlib-dev lzo-dev ncurses-dev \
+    readline-dev musl-dev lz4-dev vde2-dev
+}
```

Comment:
What are these Alpine stuff for? Just wondering.

Reply:
It's the same as for the other distributions — to build the project and run the test suite. We don't build packages for it though, because Alpine is updated very frequently anyway.

Reply:
In that case, could we provide an Alpine package please? I had a look at their repository, they don't seem to provide a package for tinc 1.1. 

This is the link I looked at: 
https://pkgs.alpinelinux.org/packages?name=tinc&branch=edge&repo=main

Reply:
It's there, but is named `tinc-pre` and is located in the `community` repository.

https://pkgs.alpinelinux.org/packages?name=tinc-pre&branch=edge&repo=community

Reply:
Ah ok. :)